WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -2955,71 +3651,57 @@ e_britannia = {
 					add = @better_than_the_alternatives_score
 				}
 			}
-			
 			c_warwickshire = {
 				color = { 190 30 30 }
 				cultural_names = {
 					#Roman Cultures
-					name_list_roman = cn_praesidium_cornaviorum
-					name_list_romano_gallic = cn_praesidium_cornaviorum
-					name_list_romano_hispanic = cn_praesidium_cornaviorum
-					name_list_romano_british = cn_praesidium_cornaviorum
-					name_list_romano_african = cn_praesidium_cornaviorum
-					name_list_romano_raetian = cn_praesidium_cornaviorum
-					name_list_romano_thracian = cn_praesidium_cornaviorum
-					name_list_romano_illyrian = cn_praesidium_cornaviorum
-					name_list_romano_pannonian = cn_praesidium_cornaviorum
-					name_list_dalmatian = cn_praesidium_cornaviorum
-					name_list_romano_noric = cn_praesidium_cornaviorum
-					name_list_romano_gothic = cn_praesidium_cornaviorum
-					name_list_romano_dacian = cn_praesidium_cornaviorum
-					name_list_romano_frank = cn_praesidium_cornaviorum
-					name_list_romano_aramean = cn_praesidium_cornaviorum
-					name_list_romano_vandalic = cn_praesidium_cornaviorum
+					name_list_roman = cn_praesidium
+					name_list_romano_gallic = cn_praesidium
+					name_list_romano_hispanic = cn_praesidium
+					name_list_romano_british = cn_praesidium
+					name_list_romano_african = cn_praesidium
+					name_list_romano_raetian = cn_praesidium
+					name_list_romano_thracian = cn_praesidium
+					name_list_romano_illyrian = cn_praesidium
+					name_list_romano_pannonian = cn_praesidium
+					name_list_dalmatian = cn_praesidium
+					name_list_romano_noric = cn_praesidium
+					name_list_romano_gothic = cn_praesidium
+					name_list_romano_dacian = cn_praesidium
+					name_list_romano_frank = cn_praesidium
+					name_list_romano_aramean = cn_praesidium
+					name_list_romano_vandalic = cn_praesidium
```

Comment:
Are you shortening this because it is too long to display? If so shouldn't Cornaviorum be retained instead? Furthermore not sure why you modify this file instead of the localisation file directly

Reply:
> Are you shortening this because it is too long to display? If so shouldn't Cornaviorum be retained instead? Furthermore not sure why you modify this file instead of the localisation file directly

I did some research on the matter, and found out that the name Praesidium cannot be applied to Warwick. From what I've learned, this Praesidium is the one mentioned in Notitia Dignitarum. I used Orbis Latinis as a source for Praesidium/Warwick connection, but they based this assumption on obsolete works. The link between Warwick and Praesidium was first suggested by William Camden in late sixteenth century. In nineteen century it was still repeated by William Smith and various other authors, but nowadays Praesidium is placed somewhere in northeastern England.
Unfortunately, its hard to pick up a replacement, as I'm not sure if there were any known Roman towns near Warwick.